packages/google_sign_in/google_sign_in_platform_interface/lib/google_sign_in_platform_interface.dart

```diff
@@ -78,27 +78,28 @@ abstract class GoogleSignInPlatform {
   ///
   /// See:
   /// https://developers.google.com/identity/sign-in/web/reference#gapiauth2initparams
-  Future<void> init(
-      {@required String hostedDomain,
-      List<String> scopes,
-      SignInOption signInOption,
-      String clientId}) async {
+  Future<void> init({
+    List<String> scopes = const <String>[],
+    SignInOption signInOption = SignInOption.standard,
+    String? hostedDomain,
```

Comment:
just to confirm, `hostedDomain` is now nullable?

Reply:
I think it has always been nullable. According to docs:

```
  /// The [hostedDomain] argument specifies a hosted domain restriction. By
  /// setting this, sign in will be restricted to accounts of the user in the
  /// specified domain. By default, the list of accounts will not be restricted.
```

The default is "do not restrict" which is null value.

Reply:
got it. I was confused by the `@required` annotation